go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/MOHC-LTD/httpshopify
 go 1.16
 
 require (
-	github.com/MOHC-LTD/shopify v1.21.0
+	github.com/MOHC-LTD/shopify v1.23.1
 	golang.org/x/time v0.0.0-20210611083556-38a9dc6acbc6
 )
+
+replace github.com/MOHC-LTD/shopify => /Users/ian/Documents/dev/shopify
```

Comment:
Assuming this is a local development addition that needs to be removed?

Reply:
This has now bee changed to the package version v1.26.0